scripts/battlefields/Waughroon_Shrine/thief_in_norg.lua

```diff
@@ -30,24 +30,24 @@ content.groups =
         mobIds =
         {
             {
-                waughroonID.mob.GAKI,
-                waughroonID.mob.GAKI + 1,
-                waughroonID.mob.GAKI + 2,
-                waughroonID.mob.GAKI + 3,
+                waughroonID.mob.GAKI[1],
```

Comment:
I'm not understanding what you're trying to do here.  This looks correct as it stands, as each group is an arena.  So Gaki + 0 .. Gaki + 4 is:
INSERT INTO `mob_spawn_points` VALUES (17367059,'Gaki','Gaki',7,-177.000,60.000,-144.000,130);
INSERT INTO `mob_spawn_points` VALUES (17367060,'Rasetsu','Rasetsu',8,-177.000,60.000,-142.000,130);
INSERT INTO `mob_spawn_points` VALUES (17367061,'Onki','Onki',9,-177.000,60.000,-140.000,130);
INSERT INTO `mob_spawn_points` VALUES (17367062,'Onibi','Onibi',10,1.000,1.000,1.000,130);
INSERT INTO `mob_spawn_points` VALUES (17367063,'Ayakashi','Ayakashi',11,0.000,0.000,0.000,0);

Reply:
I thought it was simpler to read/audit having the table of initial mob IDs for each battlefield, I don't think the offsets were wrong at all. 

The bigger issue I was having is that get first id for gaki was returning the third mob, throwing everything off

Reply:
So this really doesn't fix the problem.  The issue is that the pet is in position 4 out of 5 mobs, as opposed to the end, where I made the assumption and was incorrect.  This solution (and I'm not a fan of using gettable here), would make the spirit a required mob to kill, but one of the others not.

I have a branch up that corrects the BCNM itself, but am working to capture the actual mob positions for all three areas currently.

Reply:
If your branch fixes the entryreq I'm good to just close this PR and let you handle it.

Reply:
also fyi making the spirit required is fine because it despawns when the owner dies

Reply:
Sounds good, but just a note if converting future BCNMs: Not all pets are really "pets" with the way we currently handle things.  In this case it still is and spawned, but mob_pets always attempts to spawn the pet on master spawn, which is not always correct.  In those cases, we currently need to separate that relationship and handle the accordingly (see: AAGK, AAMR, Divine Might currently)